### PR TITLE
Add SaltStack

### PR DESCRIPTION
**Pull request by hbokh**
**https://github.com/hbokh/awesome-saltstack**

**"Salt is a new approach to infrastructure management built on a dynamic communication bus. Salt can be used for data-driven orchestration, remote execution for any infrastructure, configuration management for any app stack, and much more."  
This is a new list about Salt / SaltStack, added under a NEW section named "IT automation tools".  
A section like that was missing and other future awesome-lists (e.g. about Puppet, Chef, Ansible, etc.) can be set up under this section.**

- [x] I have read and understood the [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/contributing.md) and the [instructions for creating a list](https://github.com/sindresorhus/awesome/blob/master/create-list.md).
- [x] This pull request has a descriptive title. *(For example: `Add Node.js`)*
- The list I added
	- [x] **has been around for at least 20 days,**
	- [x] is a non-generated Markdown file in a GitHub repo,
	- [x] is not a duplicate,
	- [x] only has awesome items with descriptions *(Awesome lists are curations of the best, not of everything)*,
	- [x] contains the [awesome badge](https://github.com/sindresorhus/awesome/blob/master/awesome.md#awesome-badge) on the right side of the list heading,
	- [x] has a Table of Contents section named `Contents` as the first section,
	- [x] has an [appropriate license](https://github.com/sindresorhus/awesome/blob/master/awesome.md#choose-an-appropriate-license) *(Meaning something like CC0, NOT a code licence like MIT, BSD, Apache, etc)*,
	- [x] has [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/awesome.md#include-contribution-guidelines),
	- [x] has been checked for inconsistent formatting, spelling, and grammar *(This includes starting each link description with an uppercase character and ending it with a period `.`, and consistent naming)*,
	- [x] has been added at the bottom of the appropriate category.

Added a new section with a new awesome list "Awesome SaltStack".
Other awesome lists to be placed here in the future could be about Puppet, Chef, Ansible, etc.

### CODE REVIEW

**Comment by sindresorhus** [comment]
We only add categories when there are at least two items for it.

**Comment by sindresorhus** [comment]
I've unchecked some items that are not done. Please read the guidelines more closely.

**Comment by hbokh** [comment]
Thank you @sindresorhus for your quick review - I've made the required changes.

Nonetheless, I think it's only fair to be consistent with other awesome lists, who are showing a different markup, for example [awesome-vagrant](https://github.com/iJackUA/awesome-vagrant), [awesome-docker](https://github.com/veggiemonk/awesome-docker) and / or [awesome-kubernetes](https://github.com/ramitsurana/awesome-kubernetes).
But I guess that would be very time-extensive, right?

**Comment by davisonio** [comment]
> Nonetheless, I think it's only fair to be consistent with other awesome lists, who are showing a different markup, for example awesome-vagrant, awesome-docker and / or awesome-kubernetes.
> But I guess that would be very time-extensive, right?

Those lists were added before we became more strict on entry requirements, but yet they could do with some improvements if someone has the time.


As for this list:
- Replace the bolded text with normal text description and remove the divider at the top - they are distracting
- The contents shouldn't require a second level (below `Awesome SaltStack`)
- would move twitter feed under 'Community'

**Comment by hbokh** [comment]
Thank you. Changes applied as suggested.

File: readme.md [hunk 1]
```diff
@@ -462,6 +462,7 @@ Check out my [blog](https://blog.sindresorhus.com) and follow me on [Twitter](ht
 - [Domain-Driven Design](https://github.com/heynickc/awesome-ddd) - Software development approach for complex needs by connecting the implementation to an evolving model.
 - [Quantified Self](https://github.com/willempienaar/awesome-quantified-self) - Self-tracking through technology.
 - [Alfred Workflows](https://github.com/derimagia/awesome-alfred-workflows) - Productivity app for macOS.
+- [SaltStack](https://github.com/hbokh/awesome-saltstack)
```
Comment (sindresorhus) [inline readme.md]: Add a succinct description about SaltStack (not the list), in the same format as above.

**Comment by sindresorhus** [comment]
You need to fix the merge conflict. (That does not mean opening a new pull request)

---

Some feedback:

- Should include a succinct description of the SaltStack project at the top ([example](https://github.com/willempienaar/awesome-quantified-self)).
- Should include a project logo at the top right of the readme ([example](https://github.com/sindresorhus/awesome-electron)). (The logo should link to the project website. The logo should be high-DPI, so set it to maximum half the width of the original image.).
- Use the SVG version of the license badge. (See the one in this readme)
- `\[library-name\]\(link\)` => `[library-name](link)` Should also say `resource` and not `library-name`.

> Halite - A client-side web application interface to a running Salt infrastructure (now deprecated).

It's deprecated. There's no reason to include it in the list. Deprecated things are not awesome.

File: readme.md [hunk 2]
```diff
@@ -462,6 +462,7 @@ Check out my [blog](https://blog.sindresorhus.com) and follow me on [Twitter](ht
 - [Domain-Driven Design](https://github.com/heynickc/awesome-ddd) - Software development approach for complex needs by connecting the implementation to an evolving model.
 - [Quantified Self](https://github.com/willempienaar/awesome-quantified-self) - Self-tracking through technology.
 - [Alfred Workflows](https://github.com/derimagia/awesome-alfred-workflows) - Productivity app for macOS.
+- [SaltStack](https://github.com/hbokh/awesome-saltstack) - A configuration management system built with Python.
```
Comment (sindresorhus) [inline readme.md]: Could be simplified:

> Python-based config management system.

**Comment by davisonio** [comment]
@hbokh can you remove the preceding 'A' from the description here?

**Comment by hbokh** [comment]
@davisonio Removed the "A" from the `readme.md` in my fork  / `patch-1`.
Please forgive my ignorance, but what is going on or wrong?

**Comment by davisonio** [comment]
> @davisonio Removed the "A" from the readme.md in my fork / patch-1.
> Please forgive my ignorance, but what is going on or wrong?

Great! The issue is there are some conflicts between your branch and the master branch on this repository.

**Comment by hbokh** [comment]
@davisonio I was unable to fix this on the git-CLI (I'm open for suggestions), but somehow I did managed to get it fixed using "GitHub Desktop" and merged the remote-tracking branch 'sindresorhus/master' into patch-1.

**Pull request merged**